fluentd/configs.d/openshift/filter-k8s-record-transform.conf

```diff
@@ -1,30 +1,35 @@
 <filter kubernetes.journal.container**>
-  type record_transformer
+  @type record_transformer
   enable_ruby
   <record>
-    hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
+    # FixMe
+    # Top level hostname field; it comes from record['HOSTNAME']
+    # The index mapping has a field record['kubernetes']['hostname'];
+    # it comes from record['kubernetes']['host'] or File.open('/etc/docker-hostname')
+    # Needs to set a field inside a hash.
+    # Plus we'd better avoid the file IO.
+    hostname ${(record['kubernetes']['host'] rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
     # log is set if MESSAGE is JSON valued and has err or msg field
     message ${record['MESSAGE'] || log}
-    version 1.3.0
+    level ${record['PRIORITY']}
+    ipaddr4 "#{ENV['IPADDR4']}"
+    ipaddr6 "#{ENV['IPADDR6']}"
+    version 1.4.0
```

Comment:
@richm its not that we would directly query on `version` however it may be useful to customers if they wish to compare different versions of data for consistency.

An added benefit is it helps us get a sense of which version of Fluentd is being used